.github/workflows/acceptance-workflow.yml

```diff
@@ -43,3 +47,6 @@ jobs:
           max_attempts: 3
           timeout_minutes: 20
           command: npm run acceptancetest:${{ inputs.testfilter }}
+        env:
+          TEST_WS_SERVER: ${{ inputs.test_ws_server }}
+          SUBSCRIPTIONS_ENABLED: ${{ inputs.test_ws_server }}
```

Comment:
Do we want to use the same input parameter for `SUBSCRIPTIONS_ENABLED` as `TEST_WS_SERVER`?  I suppose at this point it doesn't matter